apostrophecms/apostrophe
# Pro 4073 image alignment

## Summary

Allow gap cursor before and after inline images and tables

## What are the specific steps to test this change?

*For example:*

1. Add a new rich-text widget
2. Add a table or an image (you can add a space, select the space to make the toolbar visible)
3. Select the image or table and using the arrow up or down key, you should see a blinking cursor before or after the image/table
4. Add some text

Note: adding a inline image in an empty rich-text without setting a caption deletes the image, there is a ticket to track this

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
